translations/messages+intl-icu.de.yml

```diff
@@ -1277,6 +1277,10 @@ error.statement.submit.nowriteaccess: "In dem Verfahren können derzeit keine St
 error.statement.submit.phase: "In der aktuellen Phase des Verfahrens können keine Stellungnahmen eingereicht werden."
 error.statement.update: "Beim Aktualisieren der Stellungnahme ist ein Fehler aufgetreten."
 error.statements.marked.submitted: "Die Stellungnahmen konnten nicht eingereicht werden!"
+error.statements.zip.export.generic.xlsx: "Die XLSX-Datei konnte dem ZIP-Archiv nicht hinzugefügt werden."
+error.statements.zip.export.generic.attachment: "Beim Hinzufügen eines Anhangs ist ein unbekannter Fehler aufgetreten."
+error.statments.zip.export.filenotfoundorreadable: "Der Anhang mit dem Hash: {hash} konnte dem ZIP-Archiv nicht hinzugefügt werden."
```

Comment:
If this is an error message displayed to the user, is the hash useful for them?

Reply:
These messages are included in a text file within the created zip file. adjusted in [8afdf43](https://github.com/demos-europe/demosplan-core/pull/2337/commits/8afdf439a2803ed7db991a32fd37abac44fb2329)

Reply:
I think you got the wrong commit, it should be https://github.com/demos-europe/demosplan-core/pull/2337/commits/2701b750cb6ffab0e2bb54700c75ff026b3aa4b3.
Thanks!

Reply:
Yes you're right! I adjusted it.